src/server/chromium/crPage.ts

```diff
@@ -845,7 +845,7 @@ class FrameSession {
 
   async _createVideoRecorder(screencastId: string, options: types.PageScreencastOptions): Promise<void> {
     assert(!this._screencastId);
-    const ffmpegPath = registry.executablePath('ffmpeg');
+    const ffmpegPath = registry.findExecutable('ffmpeg')!.executablePathIfExists();
     if (!ffmpegPath)
```

Comment:
IIUC this check was never hit before, and becomes an actual check with this patch. Cool!